google/resource_compute_image.go

```diff
@@ -106,6 +106,13 @@ func resourceComputeImage() *schema.Resource {
 				Set:      schema.HashString,
 			},
 
+			"licenses": &schema.Schema{
+				Type:     schema.TypeList,
```

Comment:
Does order matter for licenses? Is it reasonable to expect someone to try and reference each individual license using interpolations?

Reply:
Neither the [CLI](https://cloud.google.com/sdk/gcloud/reference/compute/images/create) or [API](https://godoc.org/google.golang.org/api/compute/v1#Image) docs seem to give an indication that order matters. Does your previous comment about using a Set instead of a List impact this?

Reply:
Ah, [this is nice](https://www.terraform.io/docs/extend/schemas/schema-types.html#typeset). So Google isn't expecting the order to be meaningful, but I'd say it might still be intuitive to keep this as a TypeList in case the user wants to be able to refer to it using interpolations. (I'm assuming they won't be able to do this if it's an unordered `TypeSet`). So not necessarily for compatibility reasons, but for UX reasons, might be useful to keep it as-is. Let me know if you disagree, I'm not married to it.